README.md

```diff
@@ -71,7 +71,6 @@ Apache 2.0 - See [LICENSE][license-url] for more information.
 [ci-image]: https://github.com/open-telemetry/opentelemetry-ruby/workflows/CI/badge.svg?event=push
 [contrib-repo]: https://github.com/open-telemetry/opentelemetry-ruby-contrib
 [contrib-instrumentations]: https://github.com/open-telemetry/opentelemetry-ruby-contrib/tree/main/instrumentation
-[examples-github]: https://github.com/open-telemetry/opentelemetry-ruby/tree/main/examples
```

Comment:
Probably because `[examples-github]` is not used anyway in this readme, so the markdownlint auto-correct removed it. I updated the readme to include it under Useful Links